### ISSUE TITLE
Docker image is missing libxcb-shap0-dev required by Servo

### ISSUE THREAD

**Issue by jdm** [web-platform-tests/wpt#24590]
Servo recently updated a dependency and now requires libxcb-shape0-dev. Otherwise wpt.fyi test runs fail with:
```
 0:00.85 INFO Installing browser
/home/test/web-platform-tests/_venv2/servo/servo: error while loading shared libraries: libxcb-shape.so.0: cannot open shared object file: No such file or directory
```

**Comment by foolip** [comment]
@jdm is this something you can send a PR for, or is it more involved than it seems?

**Comment by jdm** [comment]
This was fixed by #24591.
